DER/FxDerivatives/FxContracts.rdf

```diff
@@ -75,132 +75,134 @@
-				<owl:someValuesFrom rdf:resource="&fibo-der-fx-ctr;ForeignExchangeSwap"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-fx-ctr;CurrencySwap"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label xml:lang="en">foreign exchange forward outright</rdfs:label>
-		<skos:definition xml:lang="en">forward contract in the forex market that locks in the price at which an entity can buy or sell a currency on a future date. Also known as &quot;outright forward currency transaction&quot;, &quot;forward outright&quot; or &quot;FX forward&quot;.</skos:definition>
-		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">In currency forward contracts, the contract holders are obligated to buy or sell the currency at a specified price, at a specified quantity and on a specified future date. These contracts cannot be transferred. Jan 10 Review Notes Outright Forward is the term for the professional markets. Spot + Swap where Swap is 2 simultaneous transactions.</fibo-fnd-utl-av:explanatoryNote>
+		<rdfs:label xml:lang="en">currency forward outright</rdfs:label>
+		<skos:definition xml:lang="en">forward contract in the forex market that locks in the price at which an entity can buy or sell a currency on a future date</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">The holders of a currency forward are obligated to buy or sell the currency at a specified price, at a specified quantity and on a specified future date. These contracts cannot be transferred. Jan 10 Review Notes Outright Forward is the term for the professional markets. Spot + Swap where Swap is 2 simultaneous transactions.</fibo-fnd-utl-av:explanatoryNote>
+		<fibo-fnd-utl-av:synonym xml:lang="en">FX forward</fibo-fnd-utl-av:synonym>
+		<fibo-fnd-utl-av:synonym xml:lang="en">forward outright</fibo-fnd-utl-av:synonym>
+		<fibo-fnd-utl-av:synonym xml:lang="en">outright forward currency transaction</fibo-fnd-utl-av:synonym>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-der-fx-ctr;ForeignExchangeSpotContract">
+	<owl:Class rdf:about="&fibo-der-fx-ctr;CurrencySpotContract">
 		<rdfs:subClassOf rdf:resource="&fibo-der-der-sp;SpotContract"/>
-		<rdfs:subClassOf rdf:resource="&fibo-der-fx-ctr;ForeignExchangeDerivative"/>
-		<rdfs:label xml:lang="en">foreign exchange spot contract</rdfs:label>
+		<rdfs:subClassOf rdf:resource="&fibo-der-fx-ctr;CurrencyDerivative"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-fx-ctr;hasSpotExchangeRate"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;ExchangeRate"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">currency spot contract</rdfs:label>
 		<skos:definition xml:lang="en">The rate of a foreign-exchange contract for immediate delivery. Also known as &quot;benchmark rates&quot;, &quot;straightforward rates&quot; or &quot;outright rates&quot;, spot rates represent the price that a buyer expects to pay for a foreign currency in another currency.</skos:definition>
```

Comment:
This represents the contract, not the rate.
"Also known as" should be moved to synonyms.

Reply:
I've removed them and revised the definition - those are synonyms for the exchange rate in some cases, but not for the contract.  I hadn't addressed this before because this is still provisional, but have done so now.